src/Core/Grid/Query/Filter/DoctrineFilterApplicator.php

```diff
@@ -80,8 +85,67 @@ public function apply(QueryBuilder $qb, SqlFilters $filters, array $filterValues
                         $qb->setParameter($name, sprintf('%s %s', $value['to'], '23:59:59'));
                     }
 
+                    break;
+                case SqlFilters::MIN_MAX:
+                    if (!isset($value['min_field']) && !isset($value['max_field'])) {
+                        break;
+                    }
+
+                    $minFieldSqlCondition = "$sqlField >= :{$filterName}_min";
+                    $maxFieldSqlCondition = "$sqlField <= :{$filterName}_max";
+
+                    switch ($this->computeMinMaxCase($value)) {
+                        case self::CASE_BOTH_FIELDS_EXIST_NOT_EQUAL:
+                            $qb->andWhere("$minFieldSqlCondition AND $maxFieldSqlCondition");
+                            $qb->setParameter("{$filterName}_min", $value['min_field']);
+                            $qb->setParameter("{$filterName}_max", $value['max_field']);
+                            break;
+                        case self::CASE_ONLY_MIN_FIELD_EXISTS:
+                            $qb->andWhere($minFieldSqlCondition);
+                            $qb->setParameter("{$filterName}_min", $value['min_field']);
+                            break;
+                        case self::CASE_ONLY_MAX_FIELD_EXISTS:
+                            $qb->andWhere($maxFieldSqlCondition);
+                            $qb->setParameter("{$filterName}_max", $value['max_field']);
+                            break;
+                        case self::CASE_BOTH_FIELDS_ARE_EQUAL:
+                            $qb->andWhere("$sqlField = :$filterName");
+                            $qb->setParameter($filterName, $value['min_field']);
+                            break;
+                    }
                     break;
             }
         }
     }
+
+    /**
+     * @param array<string, int> $value
+     *
+     * @return int|null
+     */
+    private function computeMinMaxCase(array $value): ?int
+    {
+        $minFieldExists = isset($value['min_field']);
+        $maxFieldExists = isset($value['max_field']);
+        $bothFieldsExist = $minFieldExists && $maxFieldExists;
+        $bothFieldsAreEqual = $bothFieldsExist && $value['min_field'] === $value['max_field'];
+
+        if ($minFieldExists && $maxFieldExists && !$bothFieldsAreEqual) {
+            return self::CASE_BOTH_FIELDS_EXIST_NOT_EQUAL;
+        }
+
+        if ($minFieldExists && !$maxFieldExists) {
+            return self::CASE_ONLY_MIN_FIELD_EXISTS;
+        }
+
+        if ($maxFieldExists && !$minFieldExists) {
+            return self::CASE_ONLY_MAX_FIELD_EXISTS;
+        }
+
+        if ($bothFieldsAreEqual) {
+            return self::CASE_BOTH_FIELDS_ARE_EQUAL;
+        }
+
+        return null;
```

Comment:
Is it a default usecase we never expect to happen? If yes we can be a bit defensive with
```
throw new RuntimeException('Will never happen lol haha xD');
```